src/metabase/util/tracking.clj

```diff
@@ -0,0 +1,54 @@
+(ns metabase.util.tracking
+  (:require [clj-http.client :as client]))
+
+(def ^:private ^:const ^String google-analytics-collect-url "http://www.google-analytics.com/collect")
+
+(def ^:private ^Integer anonymous-id
+  "Generate an anonymous id. Don't worry too much about hash collisions or localhost cases, etc.
+   The goal is to be able to get a rough sense for how many different hosts are throwing a specific error/event." 
+  (hash (str (java.net.InetAddress/getLocalHost))))
+
+(defn- google-analytics-tracking-code
+  "Get the GA id for the main metabase application"
+  []
+  (require 'metabase.public-settings)
+  ((resolve 'metabase.public-settings) :ga_code))
+
+(defn- anon-tracking-enabled? 
+  "To avoid a circular reference"
+  []
+  (require 'metabase.public-settings)
+  ((resolve 'metabase.public-settings) :anon-tracking-enabled))
```

Comment:
this one was right the way you had it before. `anon-tracking-enabled` is its own function but `ga_code` is not.